File: src/Python/easy/string/problem1078.py
from typing import List

class Solution:
    def findOcurrences(self, text: str, first: str, second: str) -> List[str]:
        text = text.split()
        third_words = []
        for i in range(len(text)):
            if i + 2 < len(text):
                if text[i] == first and text[i + 1] == second:
                    third_words.append(text[i + 2])
        return third_words


text = "alice is a good girl she is a good student"; first = "a"; second = "good"

solution = Solution()
solution.findOcurrences(text, first, second)

File: src/JavaScript/easy/string/problem1078.js
/**
 * @param {string} text
 * @param {string} first
 * @param {string} second
 * @return {string[]}
 */

var findOcurrences = function (text, first, second) {
  text = text.split(" ");
  let third_words = [];
  for (let i = 0; i < text.length; i++) {
    if (i + 2 < text.length) {
      if (text[i] == first && text[i + 1] == second) {
        third_words.push(text[i + 2]);
      }
    }
  }
  return third_words
};

// let text = "alice is a good girl she is a good student", first = "a", second = "good"
// let text = "we will we will rock you", first = "we", second = "will"
let text = "ypkk lnlqhmaohv lnlqhmaohv lnlqhmaohv ypkk ypkk ypkk ypkk ypkk ypkk lnlqhmaohv lnlqhmaohv lnlqhmaohv lnlqhmaohv ypkk ypkk ypkk lnlqhmaohv lnlqhmaohv ypkk",
    first = "lnlqhmaohv", second = "ypkk"

findOcurrences(text, first, second);
